### ISSUE TITLE
Error creating machine: Maximum number of retries (5) exceeded

### ISSUE THREAD

**Issue by alexandrev** [kitematic/kitematic#1080]
I can get Kitematic working with Windows 10 build 10547 (the last one at the momoent) I can get the VM 'default' working and Running but no way Kitematic connect to it. When I try the configuration, this error comes up:

"Error creating machine: Maximum number of retries (5) exceeded You will want to check the provider to make sure the machine and associated resources were properly removed."

![image](https://cloud.githubusercontent.com/assets/947041/10263241/0a897eec-69e7-11e5-8ef9-b91a1d6a6526.png)

I try with DockerToolbox 1.8.2c and then try to update the Virtualbox to latest testbuilds and stable version but no luck. 

Any idea about this problem?

**Comment by alexandrev** [comment]
Ok, Finally I achieve a solution to my problem and I want it to post it here. Maybe this could help other person with the same problem. I've been facing different issues but the most important one was this: Somehow the Virtualbox installation and the VMWare Workstation installation was making the Virtualbox network configuration go wrong.

So, first I uninstalled the VMWare Workstation. Then I realize the latest Windows 10 build is causing that the Virtualbox Host Only adapter, is not checking the "Virtualbox NDIS6 Bridged Networking Driver" so the default machine cannot start properly.

With these two tips, I got my kitematic working!

![image](https://cloud.githubusercontent.com/assets/947041/10264623/e203222c-6a11-11e5-8c8a-056cea924217.png)
